src/resources/Strings.tsx

```diff
@@ -87,7 +87,10 @@ export const Strings = {
       note: 'For the selected revision(s), no results when compared to the base revision.',
     },
     comparisonRevisionDropdown: {
-      allRevisions: 'All revisions',
+      allRevisions: {
+        key: 'all-revisions',
```

Comment:
I don't like much that this is in Strings, because this shouldn't change even if we translate the other strings. Maybe it should just be hardcoded in the code? I don't know, I believe we can also leave it this way for now.